app/src/main/res/layout/tab_tray_grid_item.xml

```diff
@@ -2,114 +2,146 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<com.google.android.material.card.MaterialCardView xmlns:android="http://schemas.android.com/apk/res/android"
+
+<!--
+A FrameLayout here is an efficient way of having a views stack while allowing:
+    - a sufficient enough canvas for the play button to be outside translated
+    without it getting clipped by the GridLayoutManager
+    - enough padding for the primary view that needs to be displayed
+    where a border around it can be placed
+-->
+<FrameLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:layout_marginStart="8dp"
-    android:layout_marginTop="16dp"
-    android:layout_marginEnd="8dp"
-    app:cardBackgroundColor="@color/photonWhite"
-    app:cardCornerRadius="@dimen/tab_tray_grid_item_border_radius"
-    app:cardElevation="0dp"
-    app:strokeColor="@color/photonLightGrey30"
-    app:strokeWidth="1dp">
-
-    <androidx.constraintlayout.widget.ConstraintLayout
-        android:id="@+id/tab_item"
+    android:clipChildren="false"
+    android:clipToPadding="false"
+    android:importantForAccessibility="no"
+    android:padding="8dp">
```

Comment:
I think this should actually be 4dp now from comparing the before and after with this PR. I think the external border does actually add a 4dp padding even though there is a -4dp margin. Can you verify? It might be best to also present a before and after screenshot of the difference. I know prior the gap between the tabs were 16dp since they had a marginStart and marginEnd of 8dp.

Reply:
This `8dp` padding is needed here because the play/pause button will be `-8dp` externally translated.

Reply:
For the `tab_tray_grid_item_border` I've used `layout_margin="-4dp"` to ensure a 4dp space for the border to the outside of the item and then `android:padding="4dp"` to ensure the item with `layout_width="match_parent"` doesn't extend to occupy that -4dp space.
Will try checking the spacing between items. The spacing does look bigger than before.

Reply:
Indeed, the padding should have been 4dp there. Thanks!